plugins/scaffolder/src/components/ActionsPage/ActionsPage.tsx

```diff
@@ -68,6 +75,34 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+const ExamplesTable = (props: { examples: ActionExample[] }) => {
+  return (
+    <Grid container>
+      {props.examples.map(example => {
+        return (
+          <>
```

Comment:
You should have a `key` on iterated elements. May need to do longhand `<Fragment key=` here

Reply:
I used the description as the fragment key. I hope that works..

Reply:
Hm. Maybe iterate `(example, index)` instead and use the index for key